libs/perun/services/src/lib/auth.service.ts

```diff
@@ -184,24 +65,20 @@ export class AuthService {
     void this.oauthService.loadDiscoveryDocumentAndLogin();
   }
 
-  /**
-   * This method is used to handle oauth callbacks.
-   *
-   * First, it finishes the authentication and then redirects user to the url
-   * he wanted to visit.
-   *
-   */
-  handleAuthCallback(): Promise<boolean> {
-    return this.oauthService.loadDiscoveryDocumentAndTryLogin();
-  }
-
   redirectToOriginDestination(): Promise<boolean> {
     const mfaRoute = sessionStorage.getItem('mfa_route');
     if (mfaRoute) {
       return this.router.navigate([mfaRoute], { replaceUrl: true });
     }
+
```

Comment:
In this method (redirectToOriginDestination())
Is this pile of code:
    let params: string[] = [];
    if (storageParams) {
      params = storageParams.split('&');
    }
    const queryParams: Params = {};
    params.forEach((param) => {
      const elements = param.split('=');
      queryParams[elements[0]] = elements[1];
    });
useless? Because when you navigating you can just mergeQueryParams?

Also this pile of code:
if (queryParams['idpFilter']) {
      this.filterShortname = String(queryParams['idpFilter']);
    }
Is already done in the constructor of the service, so do we also need to do it here?